facades/PC/src/main/java/org/terasology/engine/splash/graphics/Shader.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * The MIT License (MIT)
```

Comment:
Seeing more of these files makes me think that yeah, maybe we should make this a quick jar file we could build and make available somewhere :-)

Maybe not right this moment with this PR, but we could write a quick issue and extract it after